pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1984,7 +1984,7 @@ func makeQueuedPodInfos(num int, timestamp time.Time) []*framework.QueuedPodInfo
 	var pInfos = make([]*framework.QueuedPodInfo, 0, num)
 	for i := 1; i <= num; i++ {
 		p := &framework.QueuedPodInfo{
-			PodInfo:   framework.NewPodInfo(st.MakePod().Name(fmt.Sprintf("test-pod-%d", i)).Namespace(fmt.Sprintf("ns%d", i)).UID(fmt.Sprintf("tp-%d", i)).Obj()),
+			PodInfo:   mustNewPodInfo(st.MakePod().Name(fmt.Sprintf("test-pod-%d", i)).Namespace(fmt.Sprintf("ns%d", i)).UID(fmt.Sprintf("tp-%d", i)).Obj()),
```

Comment:
This wrapper function(mustNewPodInfo) is created because that some constants were created using `framework.NewPodInfo`

```
var (
	TestEvent    = framework.ClusterEvent{Resource: "test"}
	NodeAllEvent = framework.ClusterEvent{Resource: framework.Node, ActionType: framework.All}
	EmptyEvent   = framework.ClusterEvent{}

	lowPriority, midPriority, highPriority = int32(0), int32(100), int32(1000)
	mediumPriority                         = (lowPriority + highPriority) / 2

	highPriorityPodInfo = framework.NewPodInfo(
		st.MakePod().Name("hpp").Namespace("ns1").UID("hppns1").Priority(highPriority).Obj(),
	)
```

If we are going to use only one wrapper function. I think I can modify like this
```
var (
	highPriorityPodInfo  *framework.Podinfo
```
```
func getHighPriorityPodInfo(t *testing.T) *framework.PodInfo {
	if highPriorityPodInfo == nil {
		highPriorityPodInfo = mustNewPodInfo(t,
			st.MakePod().Name("hpp").Namespace("ns1").UID("hppns1").Priority(highPriority).Obj(),
		)
	}
	return highPriorityPodInfo
}
```
and replace `highPriorityPodInfo` with `getHighPriorityPodInfo`.
But this behavior will create 8 more new functions.Any better solutions?

Reply:
In that case, the current implementation is good.